CatNote.BLL/Services/GenericService.cs

```diff
@@ -41,13 +42,17 @@ public async Task<TModel> Update(TModel model, CancellationToken cancellationTok
 
         var resultEntity = await _genericRepository.Update(entity, cancellationToken);
 
-        return _mapper.FromEntity(resultEntity);
+        return resultEntity == null
```

Comment:
What will happen if you pass null to FromEntity method?
Maybe it will be better to add this check into the FromEntity method